packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -133,22 +139,38 @@ export default function Sidebar({
           <Nav>
             <SidebarUserButton sidebarRendered={sidebarRendered} />
             {newSquadButtonVisible && (
-              <div className="flex">
-                <Button
-                  buttonSize={ButtonSize.Small}
-                  icon={<PlusIcon />}
-                  iconOnly={!sidebarExpanded}
-                  className={classNames(
-                    'mt-0 laptop:mt-2 mb-4 btn-primary-cabbage flex flex-1',
-                    sidebarExpanded ? 'mx-3' : 'mx-1.5',
-                  )}
-                  textPosition={
-                    sidebarExpanded ? 'justify-start' : 'justify-center'
+              <div className="flex mt-0 laptop:mt-2 mb-4">
+                <SimpleTooltip
+                  content={
+                    <>
+                      You can always
+                      <br /> open a squad here
+                    </>
                   }
-                  onClick={() => openSquadBetaModal({ origin: Origin.Sidebar })}
+                  placement="right"
+                  offset={newSquadTooltipOffset}
```

Comment:
The offset should not really be needed for this, it's not custom right?
Think the placement alone should work.

Reply:
It is different by the design.